projects/packages/redirect/tests/php/patchwork.json

```diff
@@ -0,0 +1,3 @@
+{
+	"redefinable-internals": [ "str_starts_with" ]
```

Comment:
Same, no need to mock, just polyfill in bootstrap.php or something

Reply:
👍 4eaadb626f7a0b6e834a5534b80138beac8d4ae1